modele-social/règles/salarié.yaml

```diff
@@ -159,43 +161,152 @@ contrat salarié . frais professionnels . titres-restaurant . contrôle taux emp
   formule: taux participation employeur > 60%
   description: La part employeur du titre-restaurant doit être de 60% au maximum
 
-contrat salarié . frais professionnels . indemnité kilométrique vélo:
-  icônes: 🚴
-  question: Le salarié profite-t-il de l'indemnité kilométrique vélo pour se rendre au travail ?
+contrat salarié . frais professionnels . abonnement transports publics:
+  icônes: 🚍
+  valeur: oui
+
+contrat salarié . frais professionnels . abonnement transports publics . montant:
+  titre: Abonnement aux transports publics
+  question: Quel montant le salarié dépense-t-il en abonnement aux transports publics chaque mois ?
+  unité: €/mois
+  par défaut: 0 €/mois
   description: |
-    Cette indemnité n'est pour l'instant pas obligatoire.
+    L'employeur doit prendre en charge 50% du montant dépensé par le salarié pour les transports publics lui permettant de se rendre sur son lieu de travail.
+    
+    Cette prise en charge (dans la limite des 50% du montant) est exonérée de cotisations sociales et d'impôt sur le revenu.
 
-    L'employeur a le choix de la mettre en place dans son entreprise.
+    Dans le cas d'un temps partiel, le taux de prise en charge sera le même pour un mi-temps ou plus. En dessous, le taux de prise en charge sera proportionnel.
+  références:
+    Articles R3261-1 à -10 du code du travail, version 01/01/2009: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000020080272/2009-01-01
+    Article 81 du code des impôts, version en vigueur au 31/12/2020: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000042910732/2020-12-31/
+
+contrat salarié . frais professionnels . abonnement transports publics . taux de prise en charge:
+  titre: Taux de prise en charge
+  valeur:
+    produit:
+      assiette: 
+        le minimum de:
+          - temps de travail . quotité de travail
+          - 50%
+      taux: 2 * 50%
```

Comment:
Si je comprends bien les 50% sont obligatoires et exonérés de cotisations sociales, mais l'employeur peut prendre une part plus importante, et cette part plus importante reste exonérée totalement de cotisations sociales (sauf dans le cas des salariés bénéficiant de la déduction forfaitaire spécifique).

Je ne pense pas qu'il ait besoin d'exposer une nouvelle question "quel est le taux de participation employeur ?" mais ça serait peut-être bien de l'avoir dans une variable et d'afficher "taux de participation employeur" dans les formules plutôt que la valeur littérale `50%` (surtout qu'ici dans l'expression on aurait envie de simplifier le `2 * 50%` alors qu'il est juste qu'il soit là)

Reply:
> mais l'employeur peut prendre une part plus importante, et cette part plus importante reste exonérée totalement de cotisations sociales

Est-ce que c'est ce que dénote mon implémentation?  
Dans ce cas je me suis trompé. Ce que je voulais signifier, (en mettant de côté la proratisation pour simplifier): la part prise en charge est fixée à 50% (et est exonérée socio-fiscalement).

Ou bien c'est mon interprétation de la loi qui est mauvaise?

> mais ça serait peut-être bien de l'avoir dans une variable et d'afficher "taux de participation employeur" dans les formules plutôt que la valeur littérale 50% (surtout qu'ici dans l'expression on aurait envie de simplifier le 2 * 50% alors qu'il est juste qu'il soit là)

Ok!

Reply:
Je me base https://rfpaye.grouperf.com/dictionnaire/paye/20120403142302610.html :

> L’employeur peut prendre en charge une fraction du coût de l’abonnement supérieure aux 50 % imposés par la loi. Dans ce cas, le montant total de la prise en charge ne peut être exonéré que dans la limite des frais réellement engagés. En revanche, le remboursement facultatif, au-delà du minimum de 50 %, doit être réintégré dans l’assiette des cotisations (mais pas dans la base CSG/CRDS) en cas d’application d’une déduction forfaitaire spécifique pour frais professionnels*.

La règle est un peu subtile je trouve (je ne suis pas sûr de l'avoir bien compris). Mais l'idée n'est de toute façon pas d'implémenter ce cas "prise en charge > 50%", juste d'afficher une variable "taux de prise en charge = 50%" comme tu l'as noté 👍